clang/lib/Driver/Driver.cpp

```diff
@@ -4941,9 +4950,22 @@ class OffloadingActionBuilder final {
               OutputType = types::TY_Nothing;
             A = C.MakeAction<CompileJobAction>(A, OutputType);
           }
+          // Add any of the device linking steps when -ftarget-device-link is
+          // specified. Device linking is only available for spir64_gen at this
+          // time.
+          bool isGen = TargetInfo.TC->getTriple().getSubArch() ==
```

Comment:
I think variable names should be camel case and start with a capital letter per [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly)